### PR TITLE
Mention that hot backups are not supported

### PR DESCRIPTION
**Pull request by akira-kurogane**
Hi Anastasia.

It seems some people hearing of the hot backup in PSMDB get confused and think PBM supports it too. I looked at the top page and thinkg the Feature list there is probably the best place in the whole documentation to point out that hot backup is not supported.

Whilst making the edit I got a little carried away and edited some other parts of the feature list too.

Would appreciate your opinion on these edit suggestions.

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| mongodb 3.6 | failed |
| mongodb 4.0 | failed |
| mongodb 4.2 | failed |

File: doc/source/index.rst [hunk 1]
```diff
@@ -32,13 +32,13 @@ which is no longer actively developed or supported.
 .. hlist::
    :columns: 2
 
-   - Backup and restore for both classic non-sharded replica sets and sharded clusters
+   - Logical-style backup and restore. Physical-style (a.k.a. 'hot') backup and restore not supported
```
Comment (nastena1606) [inline doc/source/index.rst:35]:
```suggestion
   - Logical backup and restore. Physical (a.k.a. 'hot') backup and restore not supported
```


I would not use "-style" here. It's not searchable by Google as compared to just "physical" and "logical backup"

**Review by nastena1606 — COMMENTED** [review]
Akira, I've left an edit suggestion. If you confirm it, will then apply and merge the PR

**Comment by akira-kurogane** [comment]
That "-style" deletion suggestion was a good one. Thanks.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| mongodb 3.6 | failed |
| mongodb 4.0 | failed |
| mongodb 4.2 | failed |

**Review by nastena1606 — APPROVED** [review]

**Pull request merged**